src/engine/game/gui/ingame/character.h

```diff
@@ -18,23 +18,105 @@
 #pragma once
 
 #include "../gui.h"
-
+#include "../../../gui/control/button.h"
+#include "../../../gui/control/label.h"
+#include "../../../gui/control/listbox.h"
+#include "../../../gui/control/slider.h"
 #include "../../../scene/node/modelnode.h"
 
 namespace reone {
 
 namespace game {
 
+class InGameMenu;
+
 class CharacterMenu : public GameGUI {
 public:
-    CharacterMenu(Game *game);
+    CharacterMenu(Game *game, InGameMenu &inGameMenu);
 
     void load() override;
     void update(float dt) override;
 
     void refreshControls();
 
 private:
+    struct Binding {
+        std::shared_ptr<gui::Label> lbl3dChar {nullptr};
```

Comment:
Initialization is not required here, since `shared_ptr` is empty by default, as opposed to a raw pointer, that would be undefined.